README.md

```diff
@@ -65,6 +65,21 @@ to work on your machine and render that beautiful docs.
   ```
 
   It should be running at Port 3000.
+  
+### Creating a pull request  
+
+- Make your edits and save
+- Run ```git add .```
+- Run ```git commit -m "Commit message goes here"```
+- Finally, run ```git push origin [your branch name goes here]```
+- Create a new pull request from your forked repository (Click the `New Pull Request` button located at the top of your repo)
+- For more detais, click [here](https://opensource.com/article/19/7/create-pull-request-github)
+
+### Steps to check out the Markdownlinting issues
+
+- The ```markdownlint``` is a static analysis tool for Node.js with a library of rules to enforce standards.
+- To install as a npm package use ```npm install markdownlint --save-dev```
```

Comment:
Please use `npm install -g markdownlint-cli` rather than some other library

Reply:
just a min

Reply:
+ Please remove the definition of markdownlint. 
+ Mention how to use it -> `markdownlint '**/*.md'`
+ Can we just use single ` instead of ```? 
+ Reference to the NPM base package is not required as well.